src/Tooling/Bit.Tooling.Templates/TodoTemplate/Docs/Publish.md

```diff
@@ -0,0 +1,6 @@
+# Deployment & Publish
+
+## Publishing to IIS
```

Comment:
Let's talk about these steps:
Switch to either blazor hybrid / web assembly
Set deployment type to either `default`, `ssr` or `pwa`
We've nothing to do with IIS here.

Reply:
Ok, Added and remove iis